docs/reference/data-management/migrate-index-allocation-filters.asciidoc

```diff
@@ -0,0 +1,33 @@
+[role="xpack"]
+[[migrate-index-allocation-filters]]
+== Migrate index allocation filters to node roles
+
+If you currently use custom node attributes and routing allocation filters to move indices
+through data tiers in a hot-warm-cold architecture, we recommend that you switch to using node roles.
+The built-in node roles enable {ilm-init} to automatically migrate indices between data tiers.
+You no longer need to explicitly invoke the allocate action in your {ilm-init} policies.
```

Comment:
I think this comes with some caveats - ie. allocate action can still exist if it only specifies `number_of_replicas`. I think we should explain that? And link to the `migrate` ILM action?